### PR TITLE
f-icons@v4.12.0 - added corporate office icon

### PR DESCRIPTION
**Pull request by dawidchar**
### Added
- `corporate-office-small.svg` icon.

CorporateOfficeSmall - Copy

### CODE REVIEW

**Comment by jamieomaguire** [comment]
I can't quite tell from the preview, but is there some unwanted whitespace that could be trimmed?

**Comment by dawidchar** [comment]
> @jamieomaguire:  I can't quite tell from the preview, but is there some unwanted whitespace that could be trimmed?

Whitespace has been removed 😄

**Review by zlatinivanov-je — DISMISSED** [review]

**Comment by zlatinivanov-je** [comment]
Did you run the file through [svgomg](https://jakearchibald.github.io/svgomg/) as per Readme

**Comment by dawidchar** [comment]
> Did you run the file through [svgomg](https://jakearchibald.github.io/svgomg/) as per Readme

Yes 😄

**Comment by jamieomaguire** [comment]
One other thing @dawidchar - is this an icon defined in the PIE design system? Where did it originate from? :)

File: icons/corporate-office.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="2 3 20 18">
```
Comment (jamieomaguire) [inline icons/corporate-office.svg:1]: I assume this was put through software to remove the whitespace like an svg timmer but the viewbox coordinates look at little strange to me. Usually the min-x and min-y (first two values) are 0. Is there a reason they're 2 and 3?

Comment (dawidchar) [reply icons/corporate-office.svg:1]: The 2 and 3 are offsets from where the icon should start, which causes it to be trimmed, however I've updated the SVG to remove viewbox trimming and the updated code should start at 0 0

Comment (jamieomaguire) [reply icons/corporate-office.svg]: awesome - thanks for explaining :)

**Review by jamieomaguire — APPROVED** [review]

**Comment by dawidchar** [comment]
> One other thing @dawidchar - is this an icon defined in the PIE design system? Where did it originate from? :)

The icon is to allow us to add a new corporate button to the header, I was passed down the icon from my product manager that I believe originated from design, however I've just raised the question to clarify 😄 

image

**Review by zlatinivanov-je — DISMISSED** [review]

**Comment by dawidchar** [comment]
I've double-checked, and that icon was indeed an old icon, I've updated the icon to be in line with the PIE design system, where this icon can also be found 😄 Sorry for the confusion.

**Comment by dawidchar** [comment]
I've got both, the 16x16 original version with a slight border and a 12x12 with no border, which is the recommended one to upload? 😄

**Review by sonnyprince — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**